src/features/shared/components/wallet-section.tsx

```diff
@@ -1,55 +1,66 @@
+import { DepositDialog } from '@/features/wallet/components/deposit-dialog';
+import { investorsMeRetrieveOptions } from '@/client/@tanstack/react-query.gen';
 
 export function WalletSection() {
-  const { t } = useTranslation();
+  const { i18n, t } = useTranslation();
+  const [depositDialogOpen, setDepositDialogOpen] = useState(false);
 
   const {
     data: accountValue,
     isPending,
     isError,
     isSuccess,
-  } = useQuery(investorsMeCurrentAccountValueRetrieveOptions());
+  } = useQuery(investorsMeRetrieveOptions());
 
   return (
-    <SidebarMenuItem className="flex items-center gap-1">
-      <SidebarMenuButton tooltip={t('common.wallet')} asChild>
-        <div className="flex items-center justify-between">
-          <Wallet />
-          {isPending && <Skeleton className="h-6 w-24" />}
-          {isError && <Skeleton className="h-6 w-24" />}
-          {isSuccess &&
-            new Intl.NumberFormat('en-US', {
-              style: 'currency',
-              currency: t('common.currency'),
-            }).format(accountValue.total_account_value)}
-          <Button
-            size="icon"
-            className="ml-auto size-8 group-data-[collapsible=icon] bg-primary active:bg-primary/90  hover:bg-primary/90 duration-200 ease-linear"
-            aria-label={t('common.add')}
-            onClick={() =>
-              toast('Wallet clicked!', {
-                duration: Infinity,
-                action: {
-                  label: 'Close',
-                  onClick: () => console.log('Close'),
-                },
-              })
-            }
-          >
-            <Plus />
-          </Button>
-        </div>
-      </SidebarMenuButton>
-    </SidebarMenuItem>
+    <>
+      <SidebarMenuItem className="flex items-center gap-1">
+        <SidebarMenuButton
+          className="active:bg-transparent hover:bg-transparent cursor-default"
+          asChild
+          tooltip={
+            isSuccess
+              ? `${t('common.wallet')}: ${toFixedLocalized(parseFloat(accountValue.balance), i18n.language, 2)}`
+              : t('common.wallet')
+          }
+        >
+          <div className="flex items-center gap-2 h-12 ml-1">
+            <Wallet />
+            {isPending && <Skeleton className="h-6 w-24" />}
+            {isError && <Skeleton className="h-6 w-24" />}
+            {isSuccess &&
+              toFixedLocalized(
+                parseFloat(accountValue.balance),
+                i18n.language,
+                2
+              )}
+          </div>
+        </SidebarMenuButton>
+        <Button
+          size="icon"
+          className="ml-auto size-8 group-data-[collapsible=icon]:bg-primary active:bg-primary/90 hover:bg-primary/90 duration-200 ease-linear"
+          aria-label={t('common.add')}
+          onClick={() => setDepositDialogOpen(true)}
+        >
+          <Plus />
+        </Button>
+      </SidebarMenuItem>
+
+      <DepositDialog
```

Comment:
sorry, can't find the deposit dialog rn, and it's more of a backend issue i guess, but when i attempt to deposit 500$ i get info that the amount must be > 0
<img width="1932" height="1186" alt="image" src="https://github.com/user-attachments/assets/ea1d8eb5-bcbc-4b82-a5cd-eb27958cab51" />